### DISCUSSION TITLE
ISO Album Art

### DISCUSSION THREAD

**Discussion by maxycatz** [jcorporation/myMPD#1243] [Q&A] [answered]
Have ISOs that are playable via myMPD/MPD, but I can't get the art to show.  I have the folder with the relevant cover.jpg in with the ISO file.

What can I change to get the art to show for these files?

**Comment by jcorporation** [comment]
Please post debug logs.

**Comment by maxycatz** [comment, marked as answer]
Am guessing I was on ver 13 when this was an issue.  Now on ver 18 and all album art for  ISO/DSD are showing.
